Assets/Shopify/UIToolkit/Test/Unit/Editor/TestShopControllerBase.cs

```diff
@@ -46,34 +46,10 @@ public class MockShopController : ShopControllerBase {
             Assert.AreSame(client, clientOnNextCall);
         }
 
-        [Test]
-        public void TestClearsCachedClientIfAccessTokenChanges() {
-            var client = _shopController.Client;
-
-            _shopController.AccessToken = "1234";
-
-            var clientOnNextCall = _shopController.Client;
-
-            Assert.AreNotSame(client, clientOnNextCall);
-            Assert.AreEqual("1234", clientOnNextCall.AccessToken);
-        }
-
-        [Test]
-        public void TestClearsCachedClientIfShopDomainChanges() {
-            var client = _shopController.Client;
-
-            _shopController.ShopDomain = "other.myshopify.com";
-
-            var clientOnNextCall = _shopController.Client;
-
-            Assert.AreNotSame(client, clientOnNextCall);
-            Assert.AreEqual("other.myshopify.com", clientOnNextCall.Domain);
-        }
-
```

Comment:
this doesn't happen anymore.... because we just invalidate the client if anything changes on the serialized object.